projects/js-packages/publicize-components/src/hooks/use-publicize-config/index.js

```diff
@@ -94,6 +94,7 @@ export default function usePublicizeConfig() {
 		numberOfSharesRemaining: sharesData.shares_remaining,
 		hasPaidPlan: !! getJetpackData()?.social?.hasPaidPlan,
 		isEnhancedPublishingEnabled: !! getJetpackData()?.social?.isEnhancedPublishingEnabled,
+		isSocialImageGeneratorEnabled: !! getJetpackData()?.social?.isSocialImageGeneratorEnabled,
```

Comment:
I feel like we could probably do something better to pluck out these values, both here and in the PHP. Something for a follow-up perhaps.

Reply:
Yeah, good idea. Should we create a task for that in our Inbox?